### DISCUSSION TITLE
Is it possible to calculate vibrational frequencies for only one molecule in a cluster?

### DISCUSSION THREAD

**Discussion by lxsep** [grimme-lab/xtb#1009] [Q&A]
I have a cluster of molecules but I only care about the frequencies of one molecule, is it possible to calculate the frequencies for only that molecule while having a cluster of explicit molecules?

**Comment by icamps** [comment]
Just thinking here... what if you add constrains and fix all the atom cluster coordinates/bonds/etc. fixed?
